lib/std/heap.zig

```diff
@@ -8,6 +8,129 @@ const c = std.c;
 const Allocator = std.mem.Allocator;
 const windows = std.os.windows;
 
+const page_size_darwin: ?comptime_int = if (builtin.os.tag.isDarwin())
+    switch (builtin.cpu.arch) {
+        .x86, .x86_64 => 4 << 10,
+        .thumb, .thumbeb, .arm, .armeb, .aarch64, .aarch64_be, .aarch64_32 => 16 << 10,
+        else => null,
+    }
+else
+    null;
+
+// -- <https://devblogs.microsoft.com/oldnewthing/20210510-00/?p=105200>
+const page_size_windows: ?comptime_int = if (builtin.os.tag == .windows and builtin.os.version_range.windows.min.isAtLeast(.xp))
+blk: {
+    break :blk switch (builtin.cpu.arch) {
+        .x86, .x86_64 => 4 << 10,
+        // SuperH => 4 << 10,
+        .mips, .mipsel, .mips64, .mips64el => 4 << 10,
```

Comment:
Windows for MIPS and PowerPC is not actually a thing; remove these. (Microsoft ported NT to a lot of architectures initially, but that's ancient history.)

Reply:
Cool.

Reply:
> Windows for MIPS and PowerPC is not actually a thing; remove these.

https://devblogs.microsoft.com/oldnewthing/20210510-00/?p=105200

This official article from 2021 lists MIPS and SuperH.

Reply:
Yes, he's referring to NT. Windows as most of us know it today has only run on x86, Itanium, and Arm, with Itanium support being more recently removed.